clang-tools-extra/clangd/tool/ClangdMain.cpp

```diff
@@ -242,13 +242,16 @@ opt<std::string> FallbackStyle{
     init(clang::format::DefaultFallbackStyle),
 };
 
-opt<bool> EnableFunctionArgSnippets{
-    "function-arg-placeholders",
-    cat(Features),
-    desc("When disabled, completions contain only parentheses for "
-         "function calls. When enabled, completions also contain "
-         "placeholders for method parameters"),
-    init(CodeCompleteOptions().EnableFunctionArgSnippets),
+opt<Config::ArgumentListsOption> PlaceholderOption{
```

Comment:
For simplicity and consistency with `--background-index`, let's keep the name and type of this unchanged